controllers/apps/transformer_cluster_deletion.go

```diff
@@ -157,7 +158,7 @@ func kindsForHalt() ([]client.ObjectList, []client.ObjectList) {
 	namespacedKinds, nonNamespacedKinds := kindsForDoNotTerminate()
 	namespacedKindsPlus := []client.ObjectList{
 		&kbappsv1.ComponentList{},
-		&appsv1alpha1.OpsRequestList{},
+		&opssv1alpha1.OpsRequestList{},
```

Comment:
Ops is on top of the cluster and operates on the cluster, so its resources should not be deleted by the cluster controller.